camelot/view/controls/editors/one2manyeditor.py

```diff
@@ -104,20 +107,41 @@ def __init__(self,
         self.set_right_toolbar_actions(kw['action_routes'])
         self.set_columns(columns)
 
+        selection_model = table.selectionModel()
+        if selection_model is not None:
+            # a queued connection, since the selection of the selection model
+            # might not be up to date at the time the currentRowChanged
+            # signal is emitted
+            selection_model.currentRowChanged.connect(
+                self.current_row_changed, type=Qt.QueuedConnection
+            )
+
+    @classmethod
+    def _register_rendered_action(cls, qobject):
+        next_rendered_action = cls._rendered_action_counter.__next__()
+        rendered_action_name = 'rendered_action_{}'.format(next_rendered_action)
+        qobject.setObjectName(rendered_action_name)
+        return rendered_action_name
+
     def render_action(self, action, parent):
         if action.render_hint == RenderHint.TOOL_BUTTON:
             # Use tool button, because this one sets the popup mode
             # to instant if there are modes in the state
-            return ActionToolbutton(action, self.gui_context, parent)
+            qobject = ActionToolbutton(action, self.gui_context, parent)
         elif action.render_hint == RenderHint.PUSH_BUTTON:
-            return ActionPushButton(action, self.gui_context, parent)
+            qobject = ActionPushButton(action, self.gui_context, parent)
         elif action.render_hint == RenderHint.COMBO_BOX:
-            return ComboBoxFilterWidget(action, self.gui_context, parent)
-        raise Exception('Unhandled render hint {} for {}'.format(action.render_hint, type(action)))
+            qobject = ComboBoxFilterWidget(action, self.gui_context, parent)
+        else:
+            raise Exception('Unhandled render hint {} for {}'.format(action.render_hint, type(action)))
+        rendered_action_name = self._register_rendered_action(qobject)
```

Comment:
ik zou hier de object name ofwel op basis vd action_route maken, ofwel de mapping doen op basis vd action route.

het action object mag in principe nt gekend zijn in de ui

Reply:
done